### PR TITLE
Bump payloads to 2.0.97

### PR DESCRIPTION
**Pull request by bwatters-r7**
Bump payloads to 2.0.97, adding https://github.com/rapid7/metasploit-payloads/pull/588

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -523,4 +523,4 @@ DEPENDENCIES
   yard
 
 BUNDLED WITH
-   2.1.4
+   2.3.22
```
Comment (smcintyre-r7) [inline Gemfile.lock:526]: I'm wondering if this change is the problem that's causing the tests to fail.

Comment (bwatters-r7) [reply Gemfile.lock:526]: Yeah; I was going to give it a bit, but that seemed like the most likely.  Not sure why that would mean that `rake` was unavailable, though.

Comment (bwatters-r7) [reply Gemfile.lock]: Yup; it looks like it was unhappy about the new bundler version..... :shrug:

**Comment by bwatters-r7** [comment]
I'm done with this PR.....

**Pull request closed without merging**